ide/che-core-ide-app/src/main/java/org/eclipse/che/ide/client/BootstrapController.java

```diff
@@ -135,6 +136,8 @@ public void execute() {
 
     /** Displays the IDE */
     private void displayIDE() {
+        FontAwesome.injectCSS();
```

Comment:
It would be much more better to inject it through start-up [Component](https://github.com/codenvy/che-core/blob/master/ide/che-core-ide-app/src/main/java/org/eclipse/che/ide/core/Component.java) like [ZeroClipboardInjector](https://github.com/codenvy/che-core/blob/master/ide/che-core-ide-app/src/main/java/org/eclipse/che/ide/bootstrap/ZeroClipboardInjector.java). It will allow to avoid hardcode it there.